flocker/common/script.py

```diff
@@ -74,6 +76,10 @@ def main(reactor, options):
 
 
 def eliot_logging_service(log_file, reactor, capture_stdout):
+    # FLOC-2538 - So log_file may be an instance of
```

Comment:
Should be fine, `LogFile` is sufficiently similar to a normal `file` that  ThreadedFileWriter will be happy.